pkg/sbom/scanner/scanner.go

```diff
@@ -124,6 +124,10 @@ func (s *Scanner) enoughDiskSpace(opts sbom.ScanOptions) error {
 // sendResult sends a ScanResult to the channel associated with the scan request.
 // This function should not be blocking
 func sendResult(requestID string, result *sbom.ScanResult, collector collectors.Collector) {
+	if result == nil {
+		_ = log.Errorf("nil result for '%s'", requestID)
```

Comment:
This `_ =` pattern shouldn’t be needed, is it ?

```suggestion
		log.Errorf("nil result for '%s'", requestID)
```

Reply:
It's not, I just gave up the fight against my linter